tests/integration_tests/hardhat/contracts/TestMessageCall.sol

```diff
@@ -0,0 +1,25 @@
+pragma solidity > 0.6.6;
```

Comment:
nit: not sure we have any purpose to use such old version

Reply:
adjust to [> 0.8.0](https://github.com/evmos/ethermint/blob/main/tests/integration_tests/hardhat/hardhat.config.ts#L10)